src/main/ionic/peers/src/app/app.module.ts

```diff
@@ -16,11 +17,13 @@ import { CardsState } from "../states/cards-state";
 import { ComponentsModule } from "../components/components.module";
 import { AppState } from "../states/app-state";
 
+
 @NgModule({
   declarations: [
     MyApp,
     HomePage,
     ListPage,
```

Comment:
Can you delete the ListPage declarations as well as the actual files?

Reply:
Yeah